### PR TITLE
feat: show number of likes

### PR DESCRIPTION
**Pull request by christian14b**
image

### CODE REVIEW

**Comment by dyersituations** [comment]
@christian14b Looks like test failures.

**Comment by roger-in-kiva** [comment]
I don't see a ticket for number of replies but I see mocks on it. Should/could we include here?

Screenshot 2024-03-01 at 16 05 16

**Comment by roger-in-kiva** [comment]
> > I don't see a ticket for number of replies but I see mocks on it. Should/could we include here?
> > Screenshot 2024-03-01 at 16 05 16
> 
> @roger-in-kiva ACK-935 is related to the comments count

Great, thanks!

File: @kiva/kv-components/vue/KvCommentsList.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
 			:user-display-name="userDisplayName"
 			:user-public-id="userPublicId"
 			:is-liked="comment.is_liked"
-			class="tw-mb-2"
+			class="tw-mb-2 tw-not-prose"
```
Comment (dyersituations) [inline @kiva/kv-components/vue/KvCommentsList.vue:13]: What does `not-prose` add here?

Comment (dyersituations) [reply @kiva/kv-components/vue/KvCommentsList.vue:13]: What issue specifically?

Comment (roger-in-kiva) [reply @kiva/kv-components/vue/KvCommentsList.vue:13]: Adding this style to every paragraph element `<p>`.

```
.tw-prose :where(p):not(:where([class~="tw-not-prose"] *)) {
    margin-top: 2rem;
    margin-bottom: 2rem;
} 
```

**Review by dyersituations — APPROVED** [review]

**Pull request merged**